scripts/split.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable no-console */
+const execa = require('execa');
+const gitUrlParse = require('git-url-parse');
+const ora = require('ora');
+const path = require('path');
+const program = require('commander');
+const { find, kebabCase } = require('lodash');
+
+const getBranchUrl = async (branch) => {
+  const { stdout: remoteUrl } = await execa.shell('git config --get remote.origin.url');
+  const gitHttpsUrl = gitUrlParse(remoteUrl).toString('https');
+  return `git+${gitHttpsUrl}#${branch}`;
+};
+
+const getCurrentBranch = () => execa.shell('git rev-parse --abbrev-ref HEAD')
```

Comment:
```suggestion
const getCurrentBranch = async () => execa.shell('git rev-parse --abbrev-ref HEAD')
```

This way we know the function returns a `Promise` ;)